tests/manage/mcg/test_object_integrity.py

```diff
@@ -3,63 +3,170 @@
-        awscli_pod.exec_cmd_on_pod(command=f'mkdir {original_dir} {result_dir}')
-        public_s3 = boto3.resource('s3', region_name=mcg_obj.region)
-        for obj in public_s3.Bucket(constants.TEST_FILES_BUCKET).objects.all():
-            logger.info(f'Downloading {obj.key} from aws test bucket')
-            awscli_pod.exec_cmd_on_pod(
-                command=f'sh -c "cd {original_dir} && '
-                f'wget https://{constants.TEST_FILES_BUCKET}.s3.'
-                f'{mcg_obj.region}.amazonaws.com/{obj.key}"'
-            )
-            downloaded_files.append(obj.key)
-
-        bucket_name = bucket_factory(1)[0].name
-
-        # Write all downloaded objects from original_dir to the MCG bucket
-        logger.info(f'Uploading all pod objects to MCG bucket')
-        bucket_path = f's3://{bucket_name}'
-        copy_cmd = f'cp --recursive {original_dir} {bucket_path}'
-        assert 'Completed' in awscli_pod.exec_cmd_on_pod(
-            command=craft_s3_command(mcg_obj, copy_cmd), out_yaml_format=False,
-            secrets=[mcg_obj.access_key_id, mcg_obj.access_key, mcg_obj.s3_endpoint]
-        ), 'Failed to Upload objects to MCG bucket'
+        bucketname = bucket_factory(1)[0].name
+        original_dir = "/original"
+        result_dir = "/result"
+        awscli_pod.exec_cmd_on_pod(command=f'mkdir {result_dir}')
+        # Retrieve a list of all objects on the test-objects bucket and
+        # downloads them to the pod
+        full_object_path = f"s3://{bucketname}"
+        downloaded_files = helpers.retrieve_test_objects_to_pod(awscli_pod,
+                                                                original_dir)
+        # Write all downloaded objects to the new bucket
+        helpers.sync_object_directory(awscli_pod, original_dir,
+                                      full_object_path, mcg_obj)
 
         # Retrieve all objects from MCG bucket to result dir in Pod
         logger.info(f'Downloading all objects from MCG bucket to awscli pod')
-        retrieve_cmd = f'cp --recursive {bucket_path} {result_dir}'
-        assert 'Completed' in awscli_pod.exec_cmd_on_pod(
-            command=craft_s3_command(mcg_obj, retrieve_cmd), out_yaml_format=False,
-            secrets=[mcg_obj.access_key_id, mcg_obj.access_key, mcg_obj.s3_endpoint]
-        ), 'Failed to Download objects from MCG bucket'
+        helpers.sync_object_directory(awscli_pod, full_object_path,
+                                      result_dir, mcg_obj)
 
         # Checksum is compared between original and result object
         for obj in downloaded_files:
             assert mcg_obj.verify_s3_object_integrity(
                 original_object_path=f'{original_dir}/{obj}',
                 result_object_path=f'{result_dir}/{obj}', awscli_pod=awscli_pod
             ), 'Checksum comparision between original and result object failed'
+
+    @pytest.mark.parametrize(
+        argnames="amount,file_type",
```

Comment:
missing space after the comma

Reply:
For the argnames, it's a string which directly follow examples from:
https://docs.pytest.org/en/latest/parametrize.html

I also don't like current format and it's confusing, but i would follow pytest examples in case change happen in behind.